auth-web/src/components/pay/eft/ShortNameAccountLink.vue

```diff
@@ -57,50 +58,79 @@
         <template #item-slot-amountOwing="{ item }">
           <span>{{ formatCurrency(item.amountOwing) }}</span>
         </template>
-        <template #item-slot-actions="{ index }">
+        <template #expanded-item="{ item }">
+          <tr class="expanded-item-row">
+            <td
+              :colspan="headers.length"
+              class="pb-5 pl-0"
+            >
+              <span class="expanded-item scheduled-item">
+                <v-icon>mdi-clock-outline</v-icon>
+                {{ formatCurrency(item.amountOwing) }} will be applied to this account today at 6:00 p.m. Pacific Time.
```

Comment:
hmm I think this is UTC, probably shifts with the time unfortunately with the CRON

Reply:
`0 2 * * * default`

Reply:
![image](https://github.com/bcgov/sbc-auth/assets/3484109/0f48c774-e483-44e3-8f12-1348a565da9f)

Reply:
it might be for example 6pm PST then 5 pm PDT

Reply:
Checked with Ethan, added in a text change to mention this